CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## Unreleased
+
+* Dynamic dispatch of nib name lookup
```

Comment:
```suggestion
* Dynamic dispatch of nib name lookup.  
```
Tip: adding two spaces at the end of the line in markdown will make the next line be actually on a next line (but in the same paragraph) when the markdown is rendered, allowing your name + link to issue to be on its own line 😉

Reply:
Actually I just saw that the last contributor forgot those 2 spaces as well so if you could get the occasion to add those 2 spaces on the "Support for Swift 5.0" entry too that would be nice 👍

Reply:
Sure thing